services/src/util/mod.rs

```diff
@@ -1,12 +1,25 @@
+pub use geoengine_datatypes::util::Identifier;
 use serde::de::Error;
+use serde::{Deserialize, Serialize};
 
-#[macro_use]
-mod identifiers;
-pub use self::identifiers::{IdResponse, Identifier};
 pub mod config;
 mod string_token;
 pub mod user_input;
 
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+pub struct IdResponse<T: Identifier> {
+    pub id: T,
+}
+
+impl<T> IdResponse<T>
+where
+    T: Identifier,
+{
+    pub fn from_id(id: T) -> Self {
+        Self { id }
+    }
+}
```

Comment:
Can't you implement `From`?

Reply:
9198d00ef59c7e0d24fc3a82cf46730c23a9aca1